src/Futhark/Actions.hs

```diff
@@ -75,7 +75,7 @@ sexpAction =
   where
     helper :: ASTLore lore => Prog lore -> IO ()
     helper prog =
-      case encode prog of
+      case encodePretty prog of
```

Comment:
@athas Would you prefer that we use the non-prettyprinted output? It's rather verbose like this.

Reply:
No, we should solve the verbosity problem directly, not by removing whitespace.